### DISCUSSION TITLE
Punctuation for U.S.

### DISCUSSION THREAD

**Discussion by SpeechC** [deepgram/community#986] [General help]
When U.S. appears in the transcription a sentence break occurs after U. and S. is its own sentence.  What setting resolves this issue?  (I have tried using the keyword setting with no joy).

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- A request ID that triggered your error or issue.

**Comment by SpeechC** [comment]
Programming language Python.  Request ID:  58d97d9a-61b3-46e6-b162-f0cbd2b433f4
________________________________

**Comment by deepgram-community** [comment]
I'm seeing an error in our logs for this request

```Error: Query parameter keywords was set to ['{keywords_encoded}']. Input should be a valid string.```
Suggestion: Use a valid value for `keywords`.

<https://developers.deepgram.com/docs/keywords|Keywords> only accept strings, this might be causing your issue

> ![avatar](https://avatars.slack-edge.com/2023-02-20/4855338233600_fad9174023ee3f640ce1_32.png) This message was sent by John Vajda from Deepgram, via our community automation.

**Reply by SpeechC** [reply]
This is not the issue at all - here is the code:

import urllib.parse
# Keywords you want to boost
keywords_list = ["U.S."]

# JSON-encode the keywords list
keywords_json = json.dumps(keywords_list)

# URL-encode the JSON string
keywords_encoded = urllib.parse.quote(keywords_json)

url = 'https://api.deepgram.com/v1/listen?model=nova-2-finance&filler_words=true&diarize=true&smart_format=true&keywords={keywords_encoded}'
